rocketpy/plots/liquid_motor_plots.py

```diff
@@ -1,3 +1,12 @@
+import copy
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.patches import Polygon
```

Comment:
I don't know if it is only an overlook or the code lacks some calls, but two imports were not being used here. The fix is on the suggestion.

```suggestion
import numpy as np
import matplotlib.pyplot as plt
```